pkg/sql/tablemetadatacache/table_metadata_updater.go

```diff
@@ -154,7 +220,11 @@ func newTableMetadataBatchUpsertQuery(batchLen int) *tableMetadataBatchUpsertQue
 // another batch.
 func (q *tableMetadataBatchUpsertQuery) resetForBatch() {
 	q.stmt.Reset()
-	q.stmt.WriteString(`
+	q.stmt.WriteString(upsertQuery)
+	q.args = q.args[:0]
+}
+
+var upsertQuery = `
```

Comment:
nit: Move this definition to the top of the file, or at least before its usage in `resetForBatch`. Can we also add a comment explaining why we need to declare this as a global var?

Reply:
Changed this back to being inline